ui/src/pages/publications/[id]/index.tsx

```diff
@@ -1,929 +1,16 @@
-    if (publicationVersion?.selfDeclaration) list.push({ title: 'Self-declaration', href: 'self-declaration' });
-    if (showRedFlags) list.push({ title: 'Red flags', href: 'red-flags' });
-
-    const sectionList = [
-        { title: 'Main text', href: 'main-text' },
-        ...list,
-        { title: 'Funders', href: 'funders' },
-        { title: 'Conflict of interest', href: 'coi' }
-    ];
-
-    const currentCoAuthor = React.useMemo(
-        () => publicationVersion?.coAuthors?.find((coAuthor) => coAuthor.linkedUser === user?.id),
-        [publicationVersion, user?.id]
-    );
-
-    const alertSeverity = React.useMemo(() => {
-        if (publicationVersion?.user?.id === user?.id) {
-            return 'INFO';
-        }
-        if (publicationVersion?.currentStatus === 'DRAFT') {
-            return 'WARNING';
-        }
-        if (currentCoAuthor?.confirmedCoAuthor) {
-            return 'SUCCESS';
-        }
-        if (!currentCoAuthor?.confirmedCoAuthor) {
-            return 'WARNING';
-        }
-        return 'INFO';
-    }, [publicationVersion, user?.id, currentCoAuthor?.confirmedCoAuthor]);
-
-    const updateCoAuthor = React.useCallback(
-        async (confirm: boolean) => {
-            setApprovalError('');
-            try {
-                await api.patch(
-                    `/publication-versions/${publicationVersion?.id}/coauthor-confirmation`,
-                    {
-                        confirm
-                    },
-                    user?.token
-                );
-            } catch (err) {
-                setApprovalError(axios.isAxiosError(err) ? err.response?.data?.message : (err as Error).message);
-            }
-        },
-        [publicationVersion?.id, user?.token]
-    );
-
-    const onBookmarkHandler = async () => {
-        if (isBookmarked) {
-            //delete the bookmark
-            try {
-                await api.destroy(`bookmarks/${bookmarkId}`, user?.token);
-                setBookmarkId(null);
-            } catch (err) {
-                console.log(err);
-            }
-        } else {
-            //create the bookmark
-            try {
-                const newBookmarkResponse = await api.post<{
-                    id: string;
-                    type: string;
-                    entityId: string;
-                    userId: string;
-                }>(
-                    `bookmarks`,
-                    {
-                        type: 'PUBLICATION',
-                        entityId: publicationVersion?.versionOf
-                    },
-                    user?.token
-                );
-
-                setBookmarkId(newBookmarkResponse.data?.id);
-            } catch (err) {
-                console.log(err);
-            }
+    return {
+        redirect: {
+            destination: `/publications/${requestedId}/versions/latest`, // this url might change in OC-391
+            permanent: false
```

Comment:
This is working fine now. The `permanent` should be set to `true` now since we release this feature and the redirect is not temporary anymore.